docs/content/en/sentry/hooks

```diff
@@ -0,0 +1,24 @@
+---
+title: Report Pre-processing (Hooks)
+description: 'Modify data before transmission'
+position: 6
+category: Sentry
+---
+
+If you would like to modify a report's captured data before transmission (e.g. add additional or scrub sensitive data), you can use a [Sentry hook](https://docs.sentry.io/platforms/javascript/configuration/options/#hooks) within your 'config'.
+
+Example of using a hook to trigger [Sentry's native User Feedback/Error Reporting Dialog](https://docs.sentry.io/platforms/javascript/guides/vue/enriching-events/user-feedback/) to gather additional information from the user:
+```
+    config: {
+      environment: process.env.NODE_ENV,
```

Comment:
Let's remove this line since it's not really relevant.

On a related note, I'm not sure if it's a good idea to control the sentry environment with **that** variable.  I'm thinking that various libraries might check it and enable more optimized/less chatty behavior for `production` and I would like my `production` and `stage` environments both to have as close behavior as possible.

So I would go for a custom ENV variable for deciding the Sentry environment.